### DISCUSSION TITLE
Onboarding PR

### DISCUSSION THREAD

**Discussion by jingkainos** [renovatebot/renovate#11580] [Request Help]
I have installed Renovate and enabled it for all of my repos, but the Onboarding PR only ran in one of the repos. Is there any condition on which type of repo will Renovate bot run on?

Thanks!

**Comment by viceice** [comment]
Checkout the logs at https://app.renovatebot.com

**Comment by rarkins** [comment]
It will only create the PR if it detects at least one dependency
